docs/Geographic_redundancy.md

```diff
@@ -15,38 +15,43 @@ The architecture of a geographically-redundant system is as follows.
 
 ![Diagram](img/Geographic_redundancy_diagram.png)
 
-Sprout has one memcached cluster per geographic region.  Although memcached
-itself does not support the concept of local and remote peers, sprout builds
+Sprout has one memcached cluster per geographic region. Although memcached
+itself does not support the concept of local and remote peers, Sprout builds
```

Comment:
I'm so pleased that my long-running orthography campaign has finally had an effect!